### PR TITLE
Added 'psetex' command in binary and cluster interfaces

### PR DESCRIPTION
**Pull request by sazzad16**
Added 'psetex' command in BinaryJedisCommands, BinaryJedisClusterCommands, JedisClusterCommands interfaces.

### CODE REVIEW

File: src/main/java/redis/clients/jedis/BinaryJedisCluster.java [hunk 1]
```diff
@@ -270,21 +270,21 @@ public Long execute(Jedis connection) {
   }
 
   @Override
-  public String psetex(final byte[] key, final long milliseconds, final byte[] value) {
```
Comment (marcosnils) [inline src/main/java/redis/clients/jedis/BinaryJedisCluster.java]: I don't understand the changes in this file. Seems like you're just renaming things but nothing is actually being added / removed?. What's the purpose of the changes here?

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedisCluster.java]: I have placed `psetex` after `setex`. Yes, this change does not have any implementation value.

But in other interfaces and classes (except this one) `setex` places earlier than `psetex`. I thought it to be a good idea to maintain it here too.

I definitely can change this if you oppose this idea.

Comment (HeartSaVioR) [reply src/main/java/redis/clients/jedis/BinaryJedisCluster.java]: I'm OK to arrange the sequence of methods, more clearly, IMHO ideally we should.

**Comment by HeartSaVioR** [comment]
LGTM

File: src/main/java/redis/clients/jedis/BinaryJedisCluster.java (hunk 1 above)
Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedisCluster.java]: It seems code re-arranging makes merging harder. So undoing those changes.

**Comment by HeartSaVioR** [comment]
I think reverting the change would make @marcosnils happier. I'll just change the status to READY_TO_MERGE.

**Comment by sazzad16** [comment]
Merged into master, 2.10.
